File: app/src/main/java/com/prashant/masterbuddy/ws/model/SelectVideosRequest.java
package com.prashant.masterbuddy.ws.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tanmay.agnihotri on 5/18/18.
 */

public class SelectVideosRequest {

    @SerializedName("startIndex")
    private int startIndex;

    @SerializedName("lastIndex")
    private int lastIndex;

    public void setStartIndex(int startIndex) {
        this.startIndex = startIndex;
    }

    public void setLastIndex(int lastIndex) {
        this.lastIndex = lastIndex;
    }
}

File: app/src/main/java/com/prashant/masterbuddy/ws/model/SelectVideosRequest.kt
package com.prashant.masterbuddy.ws.model

import com.google.gson.annotations.SerializedName

/**
 * Created by tanmay.agnihotri on 5/18/18.
 */

class SelectVideosRequest {

    @SerializedName("startIndex")
    private var startIndex: Int = 0

    @SerializedName("lastIndex")
    private var lastIndex: Int = 0

    fun setStartIndex(startIndex: Int) {
        this.startIndex = startIndex
    }

    fun setLastIndex(lastIndex: Int) {
        this.lastIndex = lastIndex
    }
}
